src/app.jsx

```diff
@@ -238,21 +238,21 @@ const NavigatorCardBody = ({ currentFilter, files, isGrid, setPath, path, sortBy
         break;
     }
 
-    const filteredFolders = filteredItems.filter((item) => (item.type === "directory"));
-    const filteredFiles = filteredItems.filter((item) => (item.type === "file"));
+    const filteredFolders = filteredItems.filter((item) => (item.type.includes("directory")));
+    const filteredFiles = filteredItems.filter((item) => (item.type.includes("file")));
     const sortedFiles = filteredFolders.sort(compare).concat(filteredFiles.sort(compare));
 
     const Item = ({ file }) => {
         return (
             <Button
               data-item={file.name} variant="plain"
               onDoubleClick={() => onDoubleClickNavigate(path, file)} onClick={() => setSelected(file)}
-              onContextMenu={(e) => { e.stopPropagation(); setSelectedContext(file) }} className={"item-button " + (file.type === "directory" ? "directory-item" : "file-item")}
+              onContextMenu={(e) => { e.stopPropagation(); setSelectedContext(file) }} className={"item-button " + (file.type.includes("directory") ? "directory-item" : "file-item")}
             >
                 <Flex direction={{ default: isGrid ? "column" : "row" }} spaceItems={{ default: isGrid ? "spaceItemsNone" : "spaceItemsMd" }}>
                     <FlexItem alignSelf={{ default: "alignSelfCenter" }}>
                         <Icon size={isGrid ? "xl" : "lg"} isInline>
-                            {file.type === "directory"
+                            {file.type.includes("directory")
```

Comment:
Why this change? there are 4 types. 'file', 'directory', 'link', 'special'
https://github.com/cockpit-project/cockpit/blob/main/src/cockpit/channels/filesystem.py#L55

Reply:
I've adjusted the code for the new bridge